LmDataset.py

```diff
@@ -1018,6 +1023,7 @@ def _data_str_to_numpy(self, vocab, s):
       unknown_label_id = vocab[self._unknown_label]
       words_idxs = [vocab.get(w, unknown_label_id) for w in words]
     return numpy.array(words_idxs, dtype=numpy.int32)
+    '''
```

Comment:
There should never be commented out code.
Why is the commented out code not correct?